boards/DCM/Src/App/states/App_DriveState.c

```diff
@@ -56,3 +70,21 @@ const struct State *App_GetDriveState(void)
 
     return &drive_state;
 }
+
+static inline bool
+    App_HasInverterFaulted(const struct DcmCanRxInterface *can_rx_interface)
+{
+    return App_CanRx_INVL_INTERNAL_STATES_GetSignal_D1_VSM_STATE_INVL(
+               can_rx_interface) ==
+               CANMSGS_INVL_INTERNAL_STATES_D1_VSM_STATE_INVL_BLINK_FAULT_CODE_STATE_CHOICE ||
+           App_CanRx_INVR_INTERNAL_STATES_GetSignal_D1_VSM_STATE_INVR(
+               can_rx_interface) ==
+               CANMSGS_INVR_INTERNAL_STATES_D1_VSM_STATE_INVR_BLINK_FAULT_CODE_STATE_CHOICE;
+}
```

Comment:
we should move this to a header file. Looks like this is the same fxn as `App_AreInvertersOK`

Reply:
Should I create a separate header file with App_AreInvertersOk() and App_HasInverterFaulted()?

Reply:
yeah, ideally a header where we can put the inlines would be nice. I noticed that there are also another section in SetPeriodicSignals that check if the AIRs are closed. Consider putting that into the header file as well

Reply:
Ok, would a header file called App_InlineChecks.h where I place all the inline checks into sound like a plan? I could also revamp `App_SetPeriodicCanSignals_TorqueRequests` to use this header file also

Reply:
I placed all the inline checks within the App_SharedStates.h file with the `static inline` declaration. I'm still a bit unsure why the static storage class is necessary but it allows the code to compile and is a common idiom based on StackOverflow and other resources https://www.greenend.org.uk/rjk/tech/inline.html

Reply:
https://stackoverflow.com/questions/47819719/static-inline-functions-in-a-header-file

Check out paragraph 2 and paragraph 4 of the answer